onnxruntime/core/mlas/lib/intrinsics/basic/min_max_elements.cpp

```diff
@@ -0,0 +1,102 @@
+/*++
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    min_max_elements.cpp
+
+Abstract:
+
+    This module implements the logic to find min and max elements with basic extension instructions.
+
+--*/
+
+#include "../mlasi.h"
+
+void
+MLASCALL
+MlasMinMaxF32Kernel(
```

Comment:
I would have just called this MlasReduceMinimumMaximumF32Kernel and put this after the MlasReduceMaximumF32Kernel in compute.cpp and not added a new file.